fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/postgres/PostgresResourceDAO.java

```diff
@@ -53,7 +53,7 @@
     private static final Logger logger = Logger.getLogger(CLASSNAME);
 
     private static final String SQL_READ_RESOURCE_TYPE = "{CALL %s.add_resource_type(?, ?)}";
-    private static final String SQL_INSERT_WITH_PARAMETERS = "{CALL %s.add_any_resource(?,?,?,?,?,?,?,?)}";
+    private static final String SQL_INSERT_WITH_PARAMETERS = "{CALL %s.add_any_resource(?,?,?,?,?,?,?,?,?,?)}";
```

Comment:
Unrelated to the current PR, but should the class-level javadoc get an update?  It says:

> This class follows the logic of the DB2 stored procedure, but does so using a series of individual JDBC statements

But I'm guessing that was a hold-over for what is now in PostgresResourceNoProcDAO.java ?

Reply:
updated the comment